src/main/webapp/js/instructorFeedbackResults.js

```diff
@@ -211,10 +211,28 @@ function bindCollapseEvents(panels, nPanels) {
             if ($(heading[0]).attr('class') === 'panel-heading') {
                 $(heading[0]).click(toggleSingleCollapse);
             }
-            $(heading[0]).attr('data-target', '#panelBodyCollapse-' + numPanels);
-            $(heading[0]).attr('id', 'panelHeading-' + numPanels);
+
+            var sectionIndex = '';
+
+            var isSectionPanel = true;
+            var sectionBody = $(heading).next().find('[id^="sectionBody-"]');
+
+            if (sectionBody.length === 0) {
+                sectionBody = $(heading).parents('[id^="sectionBody-"]');
```

Comment:
When is this branch taken? Might need to add a comment or an explaining variable here